octobot_trading/exchanges/config/ccxt_exchange_settings.py

```diff
@@ -0,0 +1,199 @@
+        enums.CCXTExchangeConfigMethods.GET_POSITION_DEFAULT.value,
+        enums.CCXTExchangeConfigMethods.GET_POSITION_BY_SUB_TYPE.value,
+        enums.CCXTExchangeConfigMethods.GET_POSITION_WITH_PRIVATE_GET_POSITION_RISK.value,
+    ]
+
+    # set default exchange config here
+
+    # overwrite classes if you need a different parser
+    ORDERS_PARSER_CLASS = parser.OrdersParser
+    TRADES_PARSER_CLASS = parser.TradesParser
+    POSITIONS_PARSER_CLASS = parser.PositionsParser
+    MARKET_STATUS_FIXER = exchange_market_status_fixer.ExchangeMarketStatusFixer
+    
+    USE_FIXED_MARKET_STATUS = None  # None -> default is with fixed market status (but without precision fixing)
+    #                                 False -> define False if not necessary
+    #                                 True -> set to True if exchange is know to need fixed market status (with precision fixing)
+
+    MARKET_STATUS_FIXER_REMOVE_PRICE_LIMITS = False
+    
+    CANDLE_LOADING_LIMIT = 0
+    MAX_RECENT_TRADES_PAGINATION_LIMIT = 0
+    MAX_ORDER_PAGINATION_LIMIT = 0
+
+    GET_ORDER_METHODS = [
+        enums.CCXTExchangeConfigMethods.GET_ORDER_DEFAULT.value,
+        enums.CCXTExchangeConfigMethods.GET_ORDER_PRIVATE.value,
+        enums.CCXTExchangeConfigMethods.GET_ORDER_FROM_OPEN_AND_CLOSED_ORDERS.value,
+        enums.CCXTExchangeConfigMethods.GET_ORDER_USING_STOP_ID.value,
+        enums.CCXTExchangeConfigMethods.GET_ORDER_FROM_TRADES.value,
+    ]
+    GET_ALL_ORDERS_METHODS = [
+        enums.CCXTExchangeConfigMethods.GET_ALL_ORDERS_DEFAULT.value,
+        enums.CCXTExchangeConfigMethods.GET_ALL_STOP_ORDERS_USING_STOP_LOSS_ENDPOINT.value,
+    ]
+    GET_OPEN_ORDERS_METHODS = [
+        enums.CCXTExchangeConfigMethods.GET_OPEN_ORDERS_DEFAULT.value,
+        enums.CCXTExchangeConfigMethods.GET_OPEN_STOP_ORDERS_USING_STOP_LOSS_ENDPOINT.value,
+    ]
+    GET_CLOSED_ORDERS_METHODS = [
+        enums.CCXTExchangeConfigMethods.GET_CLOSED_ORDERS_DEFAULT.value,
+        enums.CCXTExchangeConfigMethods.GET_CLOSED_STOP_ORDERS_USING_STOP_LOSS_ENDPOINT.value,
+    ]
+    CANCEL_ORDERS_METHODS = [
+        enums.CCXTExchangeConfigMethods.CANCEL_ORDER_DEFAULT.value,
+        enums.CCXTExchangeConfigMethods.CANCEL_STOP_ORDER_USING_STOP_LOSS_ENDPOINT.value,
+    ]
+    GET_MY_RECENT_TRADES_METHODS = [
+        enums.CCXTExchangeConfigMethods.GET_MY_RECENT_TRADES_DEFAULT.value,
+        enums.CCXTExchangeConfigMethods.GET_MY_RECENT_TRADES_USING_RECENT_TRADES.value,
+        enums.CCXTExchangeConfigMethods.GET_MY_RECENT_TRADES_USING_CLOSED_ORDERS.value,
+    ]
+    GET_POSITION_METHODS = [
+        enums.CCXTExchangeConfigMethods.GET_POSITION_DEFAULT.value,
+        enums.CCXTExchangeConfigMethods.GET_POSITION_BY_SUB_TYPE.value,
+        enums.CCXTExchangeConfigMethods.GET_POSITION_WITH_PRIVATE_GET_POSITION_RISK.value,
+    ]
+
+
+def initialize_experimental_exchange_settings(exchange, inputs):
+    experimental_enabled = exchange.UI.user_input(
```

Comment:
I think we should only enable experimental mode when DEV_MODE is activated

Reply:
currently you can only reach it throught the url /config_tentacle?name=bybit
The config wheel will NOT appear in the exchange settings.

Also fully tested exchanges will not have this experimental mode see last commit and see bybit as an example

So without checking the docs (we should add that later) nobody would notice